modules/lang-painless/src/main/java/org/elasticsearch/painless/ScriptClassInfo.java

```diff
@@ -84,8 +84,7 @@ public ScriptClassInfo(Definition definition, Class<?> baseClass) {
                 componentType -> "Painless can only implement execute methods returning a whitelisted type but [" + baseClass.getName()
                         + "#execute] returns [" + componentType.getName() + "] which isn't whitelisted.");
 
-        // Look up the argument names
-        Set<String> argumentNames = new LinkedHashSet<>();
```

Comment:
Looks ok - we only `add()` to this set and don't read it or look at the return value from `add()`.